addons/website/views/website_templates.xml

```diff
@@ -98,7 +100,7 @@
             and seo_object.website_meta_description or website_meta_description or website_meta.get('meta_description', '')"/>
         <t t-set="meta_keywords" t-value="seo_object and 'website_meta_keywords' in seo_object
             and seo_object.website_meta_keywords or website_meta_keywords"/>
-        <meta t-if="meta_description or editable" name="description" t-att-content="meta_description"/>
+        <meta name="description" t-attf-content="{{meta_description or 'Default description'}}" />
```

Comment:
- Would not t-att be better than t-attf ?
- As I am not sure about this one, could you add it in its own commit? (same PR)